### ISSUE TITLE
Detectron2 Densepose segmentation INDS

### ISSUE THREAD

**Issue by carlosimoes** [facebookresearch/detectron2#955]
## 🚀 Feature

The problem that I have is that detectron would output two images, *_IUV.png and *_INDS.png, which consists of I,U, V channels and segmented instance indices respectively. 

I already obtain the INDS and IUV, but now i wanted to identify each person in the original image with a index.

Getting the same image like in detetron:
![demo1 0001](https://user-images.githubusercontent.com/19573787/75370957-dd1dc380-58bd-11ea-8523-8305c0b1bc4f.png)
**Person 1 mask with one color and value in the original image, then person 2 with another color and value.**

## Motivation

I'm using this for a university project.

## Description
import cv2
import argparse
import glob
import logging
import os
import pickle
import sys
from typing import Any, ClassVar, Dict, List
import torch
import numpy as np

from detectron2.config import get_cfg
from detectron2.data.detection_utils import read_image
from detectron2.engine.defaults import DefaultPredictor
from detectron2.structures.boxes import BoxMode
from detectron2.structures.instances import Instances
from detectron2.utils.logger import setup_logger

from densepose.structures import DensePoseDataRelative, DensePoseOutput, DensePoseResult
from densepose import add_densepose_config
from densepose.utils.logger import verbosity_to_level
from densepose.vis.base import CompoundVisualizer
from densepose.vis.bounding_box import ScoredBoundingBoxVisualizer
from densepose.vis.densepose import (
    DensePoseResultsContourVisualizer,
    DensePoseResultsFineSegmentationVisualizer,
    DensePoseResultsUVisualizer,
    DensePoseResultsVVisualizer,
)
from densepose.vis.extractor import CompoundExtractor, create_extractor

sys.path.append("/home/carlos/detectron2/projects/DensePose")

f = open('/home/carlos/detectron2/projects/DensePose/results.pkl', 'rb') #results obtain through dump
data = pickle.load(f)
#Person 1 in image One
img_id, instance_id = 0, 0  # Look at the first image and the first detected instance

bbox_xyxy = data[img_id]['pred_boxes_XYXY'][instance_id]
result_encoded = data[img_id]['pred_densepose'].results[instance_id]
iuv_arr = DensePoseResult.decode_png_data(*result_encoded)

INDS = iuv_arr[0,:,:] #INDS_array is obtained - parts of the body
![INDS1](https://user-images.githubusercontent.com/19573787/75371779-494cf700-58bf-11ea-8fee-edf043911539.png)

**#now i want to C = np.where(INDS >= 1) == instance_id then mask this person on original image**

**Comment by ppwwyyxx** [comment]
Same as #924.
